### DISCUSSION TITLE
Repository House Keeping

### DISCUSSION THREAD

**Discussion by confused-Techie** [pulsar-edit/.github#26] [General]
Ladies, Gentlemen, and variations thereupon,

We have now forked every single repository hosted by Atom. With no discretion against if we need it, if its still being updated, archived or anything. 

---
### Archival:

Many of these repos could be archived, but we need to determine which. As some are found and archived, there will be a collection of those available within the [ORG Repo](https://github.com/pulsar-edit/.github)

---
### Rename:

Many repos us 'Atom' in the name itself. While many others additionally may use its initials. We will need to find and point those out where ever they may be.

---

If anyone has the time to spend taking a look at these issues within our collection of repos, please feel free to do so, and add the names or links of these repos to comments below for each type of repo to investigate. 

Thank you.

**Comment by confused-Techie** [comment]
# Archival:

**Reply by confused-Techie** [reply]
* [space-pencil](https://github.com/pulsar-edit/space-pencil)
* [atom-text-editor-exercise](https://github.com/pulsar-edit/atom-text-editor-exercise)
* [decoration-example](https://github.com/pulsar-edit/decoration-example)
* [ui](https://github.com/pulsar-edit/ui)
* [gsoc-atom](https://github.com/pulsar-edit/gsoc-atom)
* [space-pen-example-templates](https://github.com/pulsar-edit/space-pen-example-templates)
* [husk-sample](https://github.com/pulsar-edit/husk-sample)
* [eon](https://github.com/pulsar-edit/eon)
* [atom-theme-temp](https://github.com/pulsar-edit/atom-theme-temp)
* [nuclide-debugger-test](https://github.com/pulsar-edit/nuclide-debugger-test)
* [design-decisions](https://github.com/pulsar-edit/design-decisions)
* [atom-space-pen-views](https://github.com/pulsar-edit/atom-space-pen-views)
* [atom.io](https://github.com/pulsar-edit/atom.io)
* [atom-nightly-releases](https://github.com/pulsar-edit/atom-nightly-releases)
* [flight-manual.atom.io](https://github.com/pulsar-edit/flight-manual.atom.io)

**Reply by sertonix** [reply]
Because all language packages have been moved into the pulsar repository they should also be archived. I was a little bit confused by that

**Reply by confused-Techie** [reply]
Possibles:

* CI-test
* node
* windows-installer-1
* squeegpg
* project-ring

**Reply by sertonix** [reply]
I made a pull request on [language-c](https://github.com/pulsar-edit/language-c/pull/1) that adds and archival notice. If this is ok I will do that for every language package that is in the editor. (Not every language package is in the editor!)

**Reply by sertonix** [reply]
Atom has made a few forks without changing anything:
 * [atom-trello](https://github.com/pulsar-edit/atom-trello)
 * [react](https://github.com/pulsar-edit/react)
 * [ripgrep](https://github.com/pulsar-edit/ripgrep)
 * [slackin](https://github.com/pulsar-edit/slackin)
 * [task-lists-js](https://github.com/pulsar-edit/task-lists-js)
 * [tree-sitter](https://github.com/pulsar-edit/tree-sitter)
 * [win-iconv](https://github.com/pulsar-edit/win-iconv)

I suggest archiving or deleting them.

And some where almost nothing was changed:
 * [heroku-buildpack-multi](https://github.com/pulsar-edit/heroku-buildpack-multi)
 * [neon](https://github.com/pulsar-edit/neon)
 * [node-plist](https://github.com/pulsar-edit/node-plist)
 * [node-source-map-support](https://github.com/pulsar-edit/node-source-map-support)
 * [node-tree-sitter](https://github.com/pulsar-edit/node-tree-sitter)
 * [nsfw](https://github.com/pulsar-edit/nsfw)
 * [roaster](https://github.com/pulsar-edit/roaster)
 * [windows-installer-1](https://github.com/pulsar-edit/windows-installer-1)

Here it would be important to find if/where they are used and upgrade to the main project.

Edit: removed repos that are packages that need maintainers. They are listed [here](https://github.com/orgs/pulsar-edit/discussions/35#discussioncomment-4364450)

**Reply by confused-Techie** [reply]
@Sertonix this style of archival notice is perfect! Please feel free to add that to the other Language Repos, and if you'd like a list like you did for other repos, and I'll get through them. (Also for the last few lists, even if I haven't gotten to them don't worry I'm saving all of them and will get to them when possible)

**Reply by sertonix** [reply]
> Atom has made a few forks without changing anything:
> ...
> And some where almost nothing was changed:
> ...

I now checked with the github search function and they all seem to be unused.

**Reply by sertonix** [reply]
> @Sertonix this style of archival notice is perfect! Please feel free to add that to the other Language Repos, and if you'd like a list like you did for other repos, and I'll get through them. (Also for the last few lists, even if I haven't gotten to them don't worry I'm saving all of them and will get to them when possible)

Here your list of archival pull requests:

(Final edit: All archived!)

* ~~[language-c#1](https://github.com/pulsar-edit/language-c/pull/1)~~
* ~~[language-coffee-script#1](https://github.com/pulsar-edit/language-coffee-script/pull/1)~~
* ~~[language-csharp#1](https://github.com/pulsar-edit/language-csharp/pull/1)~~
* ~~[language-css#1](https://github.com/pulsar-edit/language-css/pull/1)~~
* ~~[language-gfm#1](https://github.com/pulsar-edit/language-gfm/pull/1)~~
* ~~[language-git#1](https://github.com/pulsar-edit/language-git/pull/1)~~
* ~~[language-go#1](https://github.com/pulsar-edit/language-go/pull/1)~~
* ~~[language-html#1](https://github.com/pulsar-edit/language-html/pull/1)~~
* ~~[language-hyperlink#1](https://github.com/pulsar-edit/language-hyperlink/pull/1)~~
* ~~[language-java#1](https://github.com/pulsar-edit/language-java/pull/1)~~
* ~~[language-javascript#1](https://github.com/pulsar-edit/language-javascript/pull/1)~~
* ~~[language-json#1](https://github.com/pulsar-edit/language-json/pull/1)~~
* ~~[language-less#1](https://github.com/pulsar-edit/language-less/pull/1)~~
* ~~[language-make#1](https://github.com/pulsar-edit/language-make/pull/1)~~
* ~~[language-mustache#1](https://github.com/pulsar-edit/language-mustache/pull/1)~~
* ~~[language-objective-c#1](https://github.com/pulsar-edit/language-objective-c/pull/1)~~
* ~~[language-perl#1](https://github.com/pulsar-edit/language-perl/pull/1)~~
* ~~[language-php#1](https://github.com/pulsar-edit/language-php/pull/1)~~
* ~~[language-property-list#1](https://github.com/pulsar-edit/language-property-list/pull/1)~~
* ~~[language-python#1](https://github.com/pulsar-edit/language-python/pull/1)~~
* ~~[language-ruby#1](https://github.com/pulsar-edit/language-ruby/pull/1)~~
* ~~[language-ruby-on-rails#1](https://github.com/pulsar-edit/language-ruby-on-rails/pull/1)~~
* ~~[language-rust#1](https://github.com/pulsar-edit/language-rust/pull/1)~~
* ~~[language-sass#1](https://github.com/pulsar-edit/language-sass/pull/1)~~
* ~~[language-shellscript#1](https://github.com/pulsar-edit/language-shellscript/pull/1)~~
* ~~[language-source#1](https://github.com/pulsar-edit/language-source/pull/1)~~
* ~~[language-sql#1](https://github.com/pulsar-edit/language-sql/pull/1)~~
* ~~[language-text#1](https://github.com/pulsar-edit/language-text/pull/1)~~
* ~~[language-todo#1](https://github.com/pulsar-edit/language-todo/pull/1)~~
* ~~[language-toml#1](https://github.com/pulsar-edit/language-toml/pull/1)~~
* ~~[language-typescript#1](https://github.com/pulsar-edit/language-typescript/pull/1)~~
* ~~[language-xml#1](https://github.com/pulsar-edit/language-xml/pull/1)~~
* ~~[language-yaml#2](https://github.com/pulsar-edit/language-yaml/pull/2)~~

**Reply by confused-Techie** [reply]
Thanks a ton for putting them all in one place, if it's fine by you, as I get through them I'll just edit your comment to cross that item out

**Reply by sertonix** [reply]
@confused-Techie
What do you think of [this](https://github.com/pulsar-edit/scroll-searcher/pull/1) pull request?
Should I mention archival there?

**Reply by confused-Techie** [reply]
> @confused-Techie What do you think of [this](https://github.com/pulsar-edit/scroll-searcher/pull/1) pull request? Should I mention archival there?

To be honest I wouldn't think it's a proper fit for archival. But maybe.

It looks like a package that a community member made and has since abandoned. But the Atom team liked it so much they decided to keep it. Now I'd recommend putting it on our [Needing Maintainers List](https://github.com/orgs/pulsar-edit/discussions/35), but otherwise am not sure if we want to put notices in the readme. 

This absolutely should be filed somewhere so that we know what this is for. But I'm not sure whats' the best approach there, having a banner on the package, something in the description, or logging it in our org wide repos document.

Might be a good idea to open a discussion about that on the Discord

**Reply by sertonix** [reply]
> > @confused-Techie What do you think of [this](https://github.com/pulsar-edit/scroll-searcher/pull/1) pull request? Should I mention archival there?
> 
> To be honest I wouldn't think it's a proper fit for archival. But maybe.
> 
> It looks like a package that a community member made and has since abandoned. But the Atom team liked it so much they decided to keep it. Now I'd recommend putting it on our [Needing Maintainers List](https://github.com/orgs/pulsar-edit/discussions/35), but otherwise am not sure if we want to put notices in the readme.
> ...

I have now added it to the [list of plugins that need maintainers](https://github.com/orgs/pulsar-edit/discussions/35#discussioncomment-4364450).

**Reply by sertonix** [reply]
Here are more archival pull requests:

* [x] [windows-installer-1#1](https://github.com/pulsar-edit/windows-installer-1/pull/1)
* [x] [vim-mode#1](https://github.com/pulsar-edit/vim-mode/pull/1)
* [x] [CI-test#1](https://github.com/pulsar-edit/CI-test/pull/1)
* [x] [squeegpg#1](https://github.com/pulsar-edit/squeegpg/pull/1)
* [x] [squeegpg-native#1](https://github.com/pulsar-edit/squeegpg-native/pull/1)
* [x] [heroku-buildpack-multi#1](https://github.com/pulsar-edit/heroku-buildpack-multi/pull/1)
* [x] [node-source-map-support#1](https://github.com/pulsar-edit/node-source-map-support/pull/1)
* [x] [node-plist#1](https://github.com/pulsar-edit/node-plist/pull/1)
* [ ] [roaster#1](https://github.com/pulsar-edit/roaster/pull/1)
* [x] [slacking#1](https://github.com/pulsar-edit/slackin/pull/1)
* [ ] [task-lists-js#1](https://github.com/pulsar-edit/task-lists-js/pull/1)
* [ ] [ripgrep#2](https://github.com/pulsar-edit/ripgrep/pull/2)
* [ ] [win-iconv#1](https://github.com/pulsar-edit/win-iconv/pull/1)
* [ ] [text-transforms#1](https://github.com/pulsar-edit/text-transforms/pull/1)
* [ ] [node-temp#1](https://github.com/pulsar-edit/node-temp/pull/1)
* [x] [coffee-cash#1](https://github.com/pulsar-edit/coffee-cash/pull/1)
* [x] [deferred-require#1](https://github.com/pulsar-edit/deferred-require/pull/1)
* [ ] [dom-listener#1](https://github.com/pulsar-edit/dom-listener/pull/1)
* [ ] [node-github-releases#1](https://github.com/pulsar-edit/node-github-releases/pull/1)
* [ ] [jasmine-waits-for-callback#1](https://github.com/pulsar-edit/jasmine-waits-for-callback/pull/1)
* [ ] [legal-eagle#1](https://github.com/pulsar-edit/legal-eagle/pull/1)
* [x] [line-length-index#1](https://github.com/pulsar-edit/line-length-index/pull/1)
* [ ] [reactionary#1](https://github.com/pulsar-edit/reactionary/pull/1)
* [ ] [highlights#1](https://github.com/pulsar-edit/highlights/pull/1)
* [ ] [spawn-as-admin#1](https://github.com/pulsar-edit/spawn-as-admin/pull/1)
* [x] [television#1](https://github.com/pulsar-edit/television/pull/1)
* [ ] [theorist#1](https://github.com/pulsar-edit/theorist/pull/1)
* [x] [jsdiff#1](https://github.com/pulsar-edit/jsdiff/pull/1)
* [ ] [grunt-coffeelint#1](https://github.com/pulsar-edit/grunt-coffeelint/pull/1)
* [ ] [cson-safe#1](https://github.com/pulsar-edit/cson-safe/pull/1)
* [x] [jasmine-reporters#1](https://github.com/pulsar-edit/jasmine-reporters/pull/1)
* [ ] [tree-sitter-typescript#1](https://github.com/pulsar-edit/tree-sitter-typescript/pull/1)
* [x] [generator-atom-npm#1](https://github.com/pulsar-edit/generator-atom-npm/pull/1)
* [ ] [etch-list-view#1](https://github.com/pulsar-edit/etch-list-view/pull/1)
* [x] [require-snapshot#1](https://github.com/pulsar-edit/require-snapshot/pull/1)
* [x] [settings-view#20](https://github.com/pulsar-edit/settings-view/pull/20)
* [x] [autocomplete-css#2](https://github.com/pulsar-edit/autocomplete-css/pull/2)
* [x] [autocomplete-html#2](https://github.com/pulsar-edit/autocomplete-html/pull/3)
* [x] [package-generator#2](https://github.com/pulsar-edit/package-generator/pull/2)
* [x] [temp-pulsar-subreddit-wiki#1](https://github.com/pulsar-edit/temp-pulsar-subreddit-wiki/pull/1)
* [x] [action-setup-pulsar#1](https://github.com/pulsar-edit/action-setup-pulsar/pull/1)
* [x] [image-view#3](https://github.com/pulsar-edit/image-view/pull/3)
* [x] [bookmarks#2](https://github.com/pulsar-edit/bookmarks/pull/2)
* [x] [archive-view#4](https://github.com/pulsar-edit/archive-view/pull/4)
* [x] [background-tips#6](https://github.com/pulsar-edit/background-tips/pull/6)
* [x] [whitespace#2](https://github.com/pulsar-edit/whitespace/pull/2)
* [x] [open-on-github#2](https://github.com/pulsar-edit/open-on-github/pull/2)
* [x] [status-bar#2](https://github.com/pulsar-edit/status-bar/pull/2)
* [x] [tabs#2](https://github.com/pulsar-edit/tabs/pull/2)
* [x] [encoding-selector#2](https://github.com/pulsar-edit/encoding-selector/pull/2)
* [x] [command-palette#2](https://github.com/pulsar-edit/command-palette/pull/2)
* [x] [autocomplete-snippets#2](https://github.com/pulsar-edit/autocomplete-snippets/pull/2)

Note: I will update this message when more are created

**Comment by confused-Techie** [comment]
# Rename:

**Reply by confused-Techie** [reply]
* [grunt-atomdoc](https://github.com/pulsar-edit/grunt-atomdoc)
* [atom-bugs](https://github.com/pulsar-edit/atom-bugs)
* [atom-trello](https://github.com/pulsar-edit/atom-trello)
* [atom-selector-linter](https://github.com/pulsar-edit/atom-selector-linter)
* [atom-light-ui](https://github.com/pulsar-edit/atom-light-ui)
* [atom-light-syntax](https://github.com/pulsar-edit/atom-light-syntax)
* [atom-dark-ui](https://github.com/pulsar-edit/atom-dark-ui)
* [atom-dark-syntax](https://github.com/pulsar-edit/atom-dark-syntax)
* [atom-babel6-transpiler](https://github.com/pulsar-edit/atom-babel6-transpiler)
* [atom-theme](https://github.com/pulsar-edit/atom-theme)
* [atom-ternjs](https://github.com/pulsar-edit/atom-ternjs)
* [atom-docker-ci](https://github.com/pulsar-edit/atom-docker-ci)
* [atom-babel7-transpiler](https://github.com/pulsar-edit/atom-babel7-transpiler)
* [atom-ui](https://github.com/pulsar-edit/atom-ui)
* [atom-languageclient](https://github.com/pulsar-edit/atom-languageclient)
* [atom-package-downloader](https://github.com/pulsar-edit/atom-package-downloader)
* [generator-atom-npm](https://github.com/pulsar-edit/generator-atom-npm)
* [atom-select-list](https://github.com/pulsar-edit/atom-select-list)
* [atom-mocha-test-runner](https://github.com/pulsar-edit/atom-mocha-test-runner)
* [atomdoc](https://github.com/pulsar-edit/atomdoc)
* [atom-keymap](https://github.com/pulsar-edit/atom-keymap)

**Reply by sertonix** [reply]
Can [this](https://github.com/pulsar-edit/atom-ide-hyperclick-1) repository be renamed to `atom-ide-hyperclick`?

I know this is not what was intended to be here but it is only a small change
